<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>511</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iDPembayaranLabel">
      <property name="text">
       <string>ID Pembayaran</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editPembayaran"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="iDTransaksiLabel">
      <property name="text">
       <string>ID Transaksi</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="editTransaksi"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="jumlahBayarLabel">
      <property name="text">
       <string>Jumlah Bayar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="tanggalBayarLabel">
      <property name="text">
       <string>Tanggal Bayar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editTBayar"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editJBayar"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>SIMPAN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>UBAH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>HAPUS</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabelPembayaran">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>531</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID PEMBAYARAN</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID TRANSAKSI</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>JUMLAH BAYAR</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>TANGGAL BAYAR</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
